class.js

```diff
@@ -1,20 +1,20 @@
-class Foo
-{
-color='negro';peso   =    34;
-  ver()
-  {
-    console.info(`Color: ${this.color}. Peso: ${this.peso}`
-    );
-// foo = 123;
-/*
-// Prueba
-test() {
-  console.info('Test');
+class Foo {
+  color = 'negro';
+  peso = 34;
+  ver() {
+    console.info(`Color: ${this.color}. Peso: ${this.peso}`);
+  }
+  test() {
+      console.info('Test');
+  }
 }
-*/
-}
-   }    class Bar   {
-  miMetodo(){if(this.imprimir){console.info('Imprimir!');}}
+
+class Bar {
+  miMetodo() {
+    if(this.imprimir) {
```

Comment:
No respeta el formato de código. En otros puntos se puede observar un espacio antes de `(`.

Recordar las siguientes reglas:

- `if`, `for`, `switch`, etc usan el mismo estilo. Por ejemplo, con un espacio luego del nombre: `if (...) {`, `switch (...) {`.
- Los métodos y funciones van siempre juntos. Por ejemplo: `console.log(...);`, `myClass.foo();`, `foo() {`.